core/src/main/java/io/ddf/analytics/AStatisticsSupporter.java

```diff
@@ -329,7 +332,7 @@ public void setThirdQuantile(double mThird_quantile) {
 
     mLog.info("Column type: " + colType);
     List<Double> pValues = Arrays.asList(percentiles);
-    Pattern p1 = Pattern.compile("(big|small|tiny|int)");
+    Pattern p1 = Pattern.compile("(big|small|tiny|int|long)");
```

Comment:
proper way to check for column type:
https://github.com/ddf-project/DDF/commit/cd446875b4a95e215e53dc385a7d96c8564fa305
@namma @mbbui